modules/grosfeld-cipher/src/grosfeld_cipher_app.cpp

```diff
@@ -0,0 +1,104 @@
+// Copyright 2021 Pronkin Dmitry
+
+#include "include/grosfeld_cipher_app.h"
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdint.h>
+#include <string.h>
+#include <string>
+#include <sstream>
+
+GronsfeldCipherApp::GronsfeldCipherApp() : message_("") {}
+
+void GronsfeldCipherApp::help(const char* appname, const char* message) {
+    message_ =
+        std::string(message) +
+          "This is a Gronsfeld cipher application.\n\n" +
+          "Please provide arguments in the following format:\n\n" +
+
+          " $" + appname + " <sourceString> <key> <operation>\n\n" +
+
+          "Where <sourceString> - string to decrypt or encrypt, " +
+          "<key> - cipher key, <operation> is one of 'decrypt' or 'encrypt'\n";
```

Comment:
Лучше использовать `\`, чтобы компилятор склеивал строки в один литерал. Типа того:
```suggestion
    message_ = std::string(message) +
        "This is a Gronsfeld cipher application.\n\n" \
        "Please provide arguments in the following format:\n\n" \
        " $" + appname + " <sourceString> <key> <operation>\n\n" \
        "Where <sourceString> - string to decrypt or encrypt, " \
        "<key> - cipher key, <operation> is one of 'decrypt' or 'encrypt'\n";
```